lib/datapacktypes/omniscript.js

```diff
@@ -10,24 +10,27 @@ var OMNI_BULK_RECORDS_LIMIT_MIN = 1000;
 OmniScript.prototype.createTitleObjects = function(input) {
     var dataPackData = input.dataPackData;
     var titleObjects = [];
-
-    if (dataPackData.VlocityDataPackData['%vlocity_namespace%__OmniScript__c'][0]['%vlocity_namespace%__Element__c']) {
-        dataPackData.VlocityDataPackData['%vlocity_namespace%__OmniScript__c'][0]['%vlocity_namespace%__Element__c'] = this.vlocity.datapacksexpand.sortList(dataPackData.VlocityDataPackData['%vlocity_namespace%__OmniScript__c'][0]['%vlocity_namespace%__Element__c'], '%vlocity_namespace%__OmniScript__c');
+    let dataField = '%vlocity_namespace%__OmniScript__c';
+    if(!dataPackData.VlocityDataPackData['%vlocity_namespace%__OmniScript__c']){
+        dataField = 'OmniScript__c';
```

Comment:
This check is good, but all the other field names should have their namespace also removed right here if this one is.